tests/pytorch/nightly/unet3d.libsonnet

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local common = import 'common.libsonnet';
+local timeouts = import 'templates/timeouts.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+local utils = import 'templates/utils.libsonnet';
+
+{
+  local command_common = |||
+    git clone https://github.com/thisisalbertliang/training.git -b test-grid-integration unet3d_test
+    pip3 install -r unet3d_test/image_segmentation/pytorch/requirements.txt
+    pip3 install tqdm
+
+    python3 unet3d_test/image_segmentation/pytorch/main.py --data_dir /datasets/kits19 \
+    --epochs 4000 \
```

Comment:
`300` should be sufficient.

Reply:
Ok. To clarify, this `--epochs` param refers to the maximum number of epochs that the training will run for.

For safety, I have lowered `--epochs` to `501` and set `--evaluate_every` to `250` and `start_eval_at` to `250`. This will give the model two chances at passing the `0.2` accuracy test.